editor/js/editor-libs/feature-detector.js

```diff
@@ -30,3 +30,16 @@ export function isDefined(feature) {
 
   return getFeatureObject(feature) !== undefined;
 }
+
+export function isMathMLSupported() {
+  // Test of Modernizr (https://github.com/Modernizr/Modernizr/blob/master/feature-detects/mathml.js) based on work by Davide (@dpvc) and David (@davidcarlisle)
+  const div = document.createElement("div");
+  div.style.position = "absolute";
+  div.style.display = "inline-block";
+  div.innerHTML = "<math><mfrac><mi>xx</mi><mi>yy</mi></mfrac></math>";
+
+  const addedElement = document.body.appendChild(div);
+  const hasMathML = div.offsetHeight > div.offsetWidth;
+  document.body.removeChild(addedElement);
+  return hasMathML;
+}
```

Comment:
I've seen this kind of mfrac-based test providing false results in the past... FYI, Yari already has a feature detection here: https://github.com/mdn/yari/blob/main/client/src/document/mathml-polyfill/index.tsx#L32 (not sure whether that can be reused somehow...)

Reply:
Thank you. I have replaced the detector code, with what is used in YARI.